src/components/Header.jsx

```diff
@@ -0,0 +1,15 @@
+import { useSelector } from 'react-redux/es/hooks/useSelector';
+
+const Header = () => {
+  const category = useSelector((state) => state.category.category.data);
+
+  return (
+    <div className="header">
+      <h1 className="title">America</h1>
+
+      <div className="season">{category[0].season}</div>
+    </div>
+  );
+};
+
+export default Header;
```

Comment:
![image](https://github.com/enocol/Metrics-webapp/assets/98453979/413f737a-5961-43ae-9cf1-8fc565a4814a)

- [ ] I really appreciate the effort you put into this PR to make the requested changes. 🙌😊 We understand that the flexibility of choosing an API may bring a slight variance in your designs, and that's why the project requirements ask you to follow these guidelines:

🌈 Colors: Select one main color for your website.
📝 Typography: Consider font face, size, and weight.
🖼️ Layout: Pay attention to composition and space between elements.

I did notice that you made changes to your navigation bar. 🧐 Please make sure it matches the original design in terms of layout. I would encourage you to add a couple of icons. 🚀😊

![image](https://github.com/enocol/Metrics-webapp/assets/98453979/3bf31e80-35db-4745-8ef6-fa0559748d60)

Keep up the good work! 🚀😃

Reply:
Hi @frank1738 You have made a recommendation for me to add a COUPLE (2) of icons. Is this correct? So that i can add the microphone and the icon on its right. Please confirm. Thanks